<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quran</class>
 <widget class="QWidget" name="Quran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-multimedia"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/quran3.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true">background-color: blue;
background-image: url(:/res/bg3.png);
font: italic 12pt &quot;Sitka&quot;;
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 700 italic 36pt &quot;Segoe UI&quot;;

color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.880682 rgba(255, 255, 255, 255), stop:0.960227 rgba(0, 0, 72, 255), stop:1 rgba(255, 255, 255, 255));
background-image: url(:/res/bg2.jpeg);</string>
         </property>
         <property name="text">
          <string>القرآن الكريم</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/bg5.png);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.8125 rgba(20, 90, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 12pt &quot;DejaVu Sans&quot;;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
